bundles/org.aposin.mergeprocessor/src/org/aposin/mergeprocessor/utils/SftpUtil.java

```diff
@@ -601,4 +533,72 @@ public String getContent(IMergeUnit mergeUnit) throws SftpUtilException {
             throw LogUtil.throwing(new SftpUtilException(message, e));
         }
     }
+
+	private static class SftpUserInfo implements UserInfo, UIKeyboardInteractive {
```

Comment:
This code looks shifted also from the rest of the file - could you fix the indentation?

Reply:
Fixed and squashed.